superset/utils/pandas_postprocessing/utils.py

```diff
@@ -101,6 +101,14 @@ def _is_multi_index_on_columns(df: DataFrame) -> bool:
     return isinstance(df.columns, pd.MultiIndex)
 
 
+def scalar_to_list(val: Any) -> Sequence[str]:
```

Comment:
nit: as we're calling this `scalar_to_list`, the return type should probably be `List[...]`:
```suggestion
def scalar_to_list(val: Any) -> List[str]:
```
Alternatively call it `scalar_to_sequence`.

Reply:
nice catch, updated.